docs/GettingStarted.md

```diff
@@ -21,10 +21,41 @@ For more comprehensive documentation please refer to the [user guide](UserGuide.
   - [Generating large datasets](#Generating-large-datasets)
   - [Next steps](#Next-steps)
 
-## Downloading the release
+## Getting DataHelix
 
-The datahelix is distributed as a zip file, with the latest release always available from the [GitHub releases page](https://github.com/finos/datahelix/releases/). You will need Java v1.8 installed to run the datahelix (you can run `java -version` to check whether you meet this requirement), it can be [downloaded here](https://www.java.com/en/download/manual.jsp).
+You can get a copy of DataHelix by one of the following means:
+- Install via Chocolatey
+- Download the zip file
+- Clone and build the project
 
+### Install via Chocolatey
+
+[Chocolatey](https://chocolatey.org/) is a Windows package manager for applications, Data Helix - since v2.1.9 - has been released as a community package.
+As such you can install the DataHelix generator by running
+
+```
+choco install datahelix --version=2.1.9
```

Comment:
This should be:

choco install datahelix --version=2.1.10

Reply:
This can be just `choco install datahelix` now